cluster-autoscaler/main.go

```diff
@@ -216,6 +216,7 @@ var (
 	maxNodeGroupBinpackingDuration     = flag.Duration("max-nodegroup-binpacking-duration", 10*time.Second, "Maximum time that will be spent in binpacking simulation for each NodeGroup.")
 	skipNodesWithSystemPods            = flag.Bool("skip-nodes-with-system-pods", true, "If true cluster autoscaler will never delete nodes with pods from kube-system (except for DaemonSet or mirror pods)")
 	skipNodesWithLocalStorage          = flag.Bool("skip-nodes-with-local-storage", true, "If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPath")
+	skipNodesWithCustomControllerPods  = flag.Bool("skip-nodes-with-custom-controller-pods", false, "If true cluster autoscaler will delete nodes with pods owned by custom controllers")
```

Comment:
The other `skip-nodes-with-` flags mean skipping during scale down: if "skip XYZ" == true, then don't delete the node. The semantics here should be the same, right now it is inverted according to the flag description.

Reply:
Sorry, you are right. I feel stupid for not noticing that. I will change the flag name.

Reply:
I am thinking of `--scale-down-node-with-custom-controller-pods`. Any other suggestions are welcome!

Reply:
Let me think about this. :pray:

Reply:
I will change the name. I hope I don't have to change it again.

Reply:
Changed the name.